config.json

```diff
@@ -0,0 +1,145 @@
+"settings": {
+  "device.ad_pubkey": "",
+  "device.custom_dns": "",
+  "device.desc": "",
```

Comment:
If we auto provisioning, it may be useful to set `device.desc` to something useful